components/stmn/Create/CreateCollection.vue

```diff
@@ -43,19 +54,119 @@
           maxlength="500"
           height="10rem" />
       </NeoField>
+
+      <NeoField
+        v-if="isLogIn"
+        variant="danger"
+        :message="balanceNotEnoughMessage">
+        <SubmitButton
+          :disabled="isMintDisabled"
+          expanded
+          label="create collection"
+          :loading="isLoading"
+          @click="submit" />
+      </NeoField>
     </form>
   </div>
 </template>
 
 <script lang="ts" setup>
-import { NeoField, NeoInput } from '@kodadot1/brick'
+import { NeoField, NeoInput, NeoSwitch } from '@kodadot1/brick'
+import { Interaction } from '@kodadot1/minimark/v1'
+import { notificationTypes, showNotification } from '@/utils/notification'
+import { CollectionToMintStatmine } from '@/composables/transaction/types'
+import SubmitButton from '@/components/base/SubmitButton.vue'
+import useLoader from '@/composables/useLoader'
```

Comment:
```suggestion
```

all composables are auto-imported

Reply:
If you are using VSCODE

![Screenshot 2023-06-26 at 18 25 30](https://github.com/kodadot/nft-gallery/assets/22471030/e1a8de54-8b97-4cc3-ac68-b68c17df604d)

And sometimes it is a good option to close IDE

Reply:
thanks viki, i have Volar installed. restart IDE should do the trick i guess

Reply:
done